server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -329,6 +343,7 @@ export class TenantManager {
 						);
 					}
 				}
+				this.logInvalidTokenJti(tenantId, token, error.code);
```

Comment:
I think this should be moved up to just below isNetworkError check. Otherwise, this could be skipped by the bypass cache block? Although, in that case, I guess it would be logged by the retry